src/tests/ftest/util/test_utils.py

```diff
@@ -888,6 +888,9 @@ def __init__(self, pool, cb_handler=None):
         self.dkey_size = BasicParameter(None)
         self.data_size = BasicParameter(None)
```

Comment:
Can self.co_prop be defined here and then you don't need to do a get.params in ior_test_base?
        self.co_prop = BasicParameter(None)
```

Reply:
Unfortunately, no... This is possible, if the DaosContainer.create uses a multiple input parameters. Presently, we are trying to create a structure which is used by DaosContainer.create. So we have to some how get the structure value (which is under daos_api.py) and manipulate it from ftest code.